### ISSUE TITLE
resultQuery input doesnt handle parameters.

### ISSUE THREAD

**Issue by checkiecheck** [frankframework/frankframework#5364]
https://github.com/ibissource/iaf/blob/45f585252fe84b0713136c536bea024316fc925a/core/src/main/java/nl/nn/adapterframework/jdbc/JdbcQuerySenderBase.java#L809

using ?{} parameter in resultQuery restults in hard error while executing with error like 
Syntax error in SQL statement "SELECT IDENTIFICATIE FROM IDENTIFICATIE WHERE TYPE = ?{IdentificatieType}[*]" [42000-214]

replacing with hardcoded value makes the query work.

**Comment by checkiecheck** [comment]
any parameter.

**Comment by thomaspj10** [comment]
You didn't specify a version, so I tested on master. I think that you forget to set the attribute `useNamedParams` to true. I was able to reproduce the error message provided above using an unit test with the same situation, so I will close this issue. The error message is fair, because there is indeed a syntax error. You forgot to enable `useNamedParams`, which caused `?{IdentificatieType}` to not be substituted.
